### PR TITLE
Hamburger Icon and MENU Label Visibility Bug

### PR DESCRIPTION
**Pull request by shaikahmadnawaz**
**Title**

"[Footer] Use stellarnav.js instead of stellarnav.min.js in production"

**Description**

This PR addresses issue #1104 

The current implementation of the footer in production uses "/stellarnav.min.js", which causes issues with visibility of the MENU label. This PR modifies the environment conditional in the "footer.html" file to use "/js/stellarnav.js" instead. Alternatively, we can make `menuLabel: ""` from the "/stellarnav.min.js" file before this PR merge, which would also resolve the issue.

[preview](https://deploy-preview-1107--mesheryio-preview.netlify.app/)

After merging this PR, the navbar will look like the screenshot attached:

Screenshot (608)

**Notes for Reviewers**

Please review the changes made in the "footer.html" file and verify that they correctly modify the environment conditional to use "/js/stellarnav.js". Also, please ensure that the proposed alternative approach of modifying `menuLabel` in the "/stellarnav.min.js" file is a viable solution.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ x ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Chadha93** [comment]
It looks like this to me in Preview. 
image

**Comment by shaikahmadnawaz** [comment]
> It looks like this to me in Preview. image

I did for mobile view

**Comment by shaikahmadnawaz** [comment]
> It looks like this to me in Preview. image

It is coming like that until the page fully loaded, after loading you can see the preview like this 👇

Screenshot (609)

**Comment by shaikahmadnawaz** [comment]
**Updated the code** and this is the screenshot of the deploy [preview link](https://deploy-preview-1107--mesheryio-preview.netlify.app/)  in mobile mode of hamburger menu.

Screenshot (610)

**Review by Udit-takkar — CHANGES_REQUESTED** [review]
Thanks for contributing @shaikahmadnawaz 

1) while clicking on hamburger icon the complete box shouldn't be blue.

https://user-images.githubusercontent.com/53316345/230572119-d51ba9d1-0d9a-4627-92c6-5428d8df83d9.mov

2) when the dropdown menu is open we should display a cross icon instead of hamburger icon with some animation. you can see https://layer5.io/ for reference.

File: _includes/footer.html [hunk 1]
```diff
@@ -46,7 +46,7 @@ <h2><a {% if column.new_window %}target="_blank" {% endif %}
 </footer>
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
 {% if jekyll.environment == 'production' %}
-<script src="{{ site.baseurl }}/js/stellarnav.min.js"></script>
```
Comment (Udit-takkar) [inline _includes/footer.html:49]: i don't think this change is correct. in production environment we should use .min.js file because they are minified version of js file

Comment (Udit-takkar) [reply _includes/footer.html:49]: stellarnav.min.js is auto generated from stellarnave.js.

**Comment by shaikahmadnawaz** [comment]
Okay I will work on this updates

**Comment by shaikahmadnawaz** [comment]
> Thanks for contributing @shaikahmadnawaz
> 
> 1. while clicking on hamburger icon the complete box shouldn't be blue.
> 
> 2. when the dropdown menu is open we should display a cross icon instead of hamburger icon with some animation. you can see https://layer5.io/ for reference.

Okay I will work on these updates

**Comment by Udit-takkar** [comment]
@shaikahmadnawaz yeah your solution is correct but these changes are required for better experience.

**Comment by shaikahmadnawaz** [comment]
> @shaikahmadnawaz yeah your solution is correct but these changes are required for better experience.

Yes, updating them also 😊

**Comment by vishalvivekm** [comment]
> > @shaikahmadnawaz yeah your solution is correct but these changes are required for better experience.
> 
> Yes, updating them also 😊

@shaikahmadnawaz Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

**Comment by shaikahmadnawaz** [comment]
> > > @shaikahmadnawaz yeah your solution is correct but these changes are required for better experience.
> > 
> > 
> > Yes, updating them also 😊
> 
> @shaikahmadnawaz Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

Someone already added it, I think it's you.

**Comment by vishalvivekm** [comment]
> > > > @shaikahmadnawaz yeah your solution is correct but these changes are required for better experience.
> > > 
> > > 
> > > Yes, updating them also 😊
> > 
> > 
> > @shaikahmadnawaz Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)
> 
> Someone already added it, I think it's you.

Hi @shaikahmadnawaz, let's discuss this in the next Websites call on 17th April 5:30 PM IST: [meeting minutes ](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#)

**Review by GaganpreetKaurKalsi — COMMENTED** [review]
Hey @shaikahmadnawaz, instead of updating the whole `stellarnav.min.js`, can't we just remove the `menuLabel:"Menu"` from the `stellarnav.min.js` file or set it to empty string? That should work.

**Review by shaikahmadnawaz — COMMENTED** [review]
Check once @GaganpreetKaurKalsi, `menuLabel:"Menu"` is set to empty string

**Comment by GaganpreetKaurKalsi** [comment]
> Check once @GaganpreetKaurKalsi, `menuLabel:"Menu"` is set to empty string

Yeah, that's true. What I am saying is that currently under Files changed tab, the whole file contents are updated(surely the `menuLabel: ""` ). The changes include formatting changes(spaces, newLine, semi-colon, etc.) for readability purposes and setting `menuLabel: ""`. Minified files should not have any formatting changes. 

Just update the menuLabel value, `menuLabel: ""`. Don't include any formatting related changes. You can disable the prettier if it is troubling you.

**Comment by shaikahmadnawaz** [comment]
> > Check once @GaganpreetKaurKalsi, `menuLabel:"Menu"` is set to empty string
> 
> Yeah, that's true. What I am saying is that currently under Files changed tab, the whole file contents are updated(surely the `menuLabel: ""` ). The changes include formatting changes(spaces, newLine, semi-colon, etc.) for readability purposes and setting `menuLabel: ""`. Minified files should not have any formatting changes.
> 
> Just update the menuLabel value, `menuLabel: ""`. Don't include any formatting related changes. You can disable the prettier if it is troubling you.

Okay, got it 👍

**Comment by Savio629** [comment]
Any reason @shaikahmadnawaz for closing this pr?

**Comment by shaikahmadnawaz** [comment]
I raised new PR.
